File: Javascrit_Capa/For/Ex4.js
// Escriba un for loop que vaya de 0 a 10. Para cada iteracion del ciclo
// multiplicara el numero por 9 y mostrará el resultado.
// por ejemplo "2 * 9 = 18"

var prompt = require('prompt-sync')()
    console.log('Tabla de multiplicar del 9 \n')

var multiplicador = 9

for (i = 0; i <= 10; i++) {
    resultado = multiplicador * i
    console.log(multiplicador + ' * ' + i + ' = ' + resultado)    
}

File: Python/If_Else/Ex4.py
# Ejemplo de elif para numero multiplo de dos

numero = int(input("Escriba un numero: "))
if numero % 2 == 0 and numero % 4 != 0:
    print(f"{numero} es multiplo de dos")
elif numero % 4 == 0:
    print(f"{numero} es multiplo de cuatro y de dos")
else:
    print(f"{numero} no es multiplo de dos")
